internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -230,6 +231,26 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"serverless_compute": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"custom_subnet_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: networkValidate.SubnetID,
+						},
+						"no_public_ip_enabled": {
```

Comment:
Prefixing a boolean with `no_` violates the [naming standard we use for bools](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md#property-naming). This needs to become  
```suggestion
						"public_ip_enabled": {
```

Reply:
Can we keep this to `no_public_ip_enabled`? If renamed to `public_ip_enabled` we need to reverse the value before send to service or set to state.  Also service will return the validation error such like: `ValidationError: Not supported to update ServerlessComputeNoPublicIP from false to true when public network access is enabled` and this validation may changed in the future.  Users may confused by the error message because the name is not same to `public_ip_enabled`

Reply:
We have many instances where we flip the boolean values around in the provider to conform to the standards the provider has. In addition that error message indicates a limitation on when `public_ip_enabled` can be updated which means we should probably have some validation around this in the Create and Update methods and can thus return our own error message instead of relying on the API to error.

Given the above this property still needs to become `public_ip_enabled`.

Reply:
ok will update it